tests/test_wps_generate_prsn.py

```diff
@@ -1,137 +1,150 @@
 import pytest
+import re
 
 from pywps import Service
 from pywps.tests import assert_response_success
 
 from .common import TESTDATA, run_wps_process
 from thunderbird.processes.wps_generate_prsn import GeneratePrsn
 
+local_test_files = TESTDATA["test_local_nc"]
+# pr, tasmin and tasmax files to make vaild sets of test_data
+local_pr_files = [f for f in local_test_files if "pr" in f]
+local_tasmin_files = [f for f in local_test_files if "tasmin" in f]
+local_tasmax_files = [f for f in local_test_files if "tasmax" in f]
 
-def build_params(kwargs):
-    if (
-        "chunk_size" in kwargs.keys() and "output_file" in kwargs.keys()
-    ):  # Not using default values
+# creating sets of test_data with the matching files
+netcdf_sets = [
```

Comment:
Generally, we don't want to include the datatype in the variable name.  Something like `netcdfs` or `netcdf_trio` may be more appropriate here.